test/ClientCDFObjectTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Acquia\ContentHubClient\test;
+
+
+use Acquia\ContentHubClient\CDF\CDFObject;
+use Acquia\ContentHubClient\CDF\ClientCDFObject;
+use Acquia\ContentHubClient\CDFAttribute;
+use PHPUnit\Framework\TestCase;
+
+class ClientCDFObjectTest extends TestCase
+{
+  /**
+   * @var ClientCDFObject
+   */
+  private $clientCdfObject;
+
+  /**
+   *
+   */
+  public function setUp() : void
+  {
+    parent::setUp();
+    $settingsData = $this->getSettingsData();
+    try {
+      $this->clientCdfObject = ClientCDFObject::create(
+        'client_cdf_id_1',
+        $settingsData
+      );
+    } catch (\Exception $exception) {
```

Comment:
@dougan88 What is the purpose of such a catch?

Reply:
@jabxjab the reason why this is made like that is because phpstorm highlights it as an unhandled exception. Although could print an error message here, but in my opinion this is optional since we're only going to run this in dev environment.

Reply:
@dougan88 Please use `@throws \Exception` phpdoc instead

Reply:
switched to @throws

Reply:
switched to \@throws

Reply:
switched to `@throws`